exercises/poker/.meta/src/reference/java/Hand.java

```diff
@@ -127,13 +127,13 @@ private int calculatedScore(Map<Integer, Integer> frequencyMap, List<Card> cards
             return 300 + triplet + Collections.max(uniqueCards);
         }
         if (rankCounts.equals(Arrays.asList(2, 2, 1))) {
-            int products_of_frequency_and_value = 0;
+            int ProductsOfFrequencyAndValue = 0;
```

Comment:
Minor nitpick: in Java it's considered best practice to name variables starting with a lowercase letter and only name classes starting with an uppercase letter 🙂